packages/core/admin/_internal/node/core/plugins.ts

```diff
@@ -167,19 +165,10 @@ const getMapOfPluginsWithAdmin = (
         throw err;
       }
     })
-    .map((plugin) => {
-      const systemPath = plugin.isLocal
-        ? path.relative(runtimeDir, plugin.pathToPlugin.split('/').join(path.sep))
-        : undefined;
-      const modulePath = systemPath ? systemPath.split(path.sep).join('/') : undefined;
-
-      return {
-        path: !plugin.isLocal
-          ? `${plugin.pathToPlugin}/strapi-admin`
-          : `${modulePath}/strapi-admin`,
-        name: plugin.name,
-        importName: camelCase(plugin.name),
-      };
-    });
+    .map((plugin) => ({
+      path: `${plugin.pathToPlugin}/strapi-admin`,
```

Comment:
Do you need the os specific separator here too?

Reply:
no because these are module paths which always use the `/`